app/components/email-input.hbs

```diff
@@ -67,7 +67,7 @@
         </button>
       </div>
     </div>
-    {{#if this.emailNotVerified }}
+    {{#if (and @user.email (not @user.email_verified))}}
```

Comment:
`isEmailVerified` is misleading though, since we need the inverse here, but only if an `email` has already been set. I haven't been able to come up with good name for that yet 🤔